torch/_export/serde/serialize.py

```diff
@@ -1496,9 +1496,19 @@ def deserialize(
                 shape_env=self.shape_env,
             )
             self.symbol_name_to_symbol: Dict[str, sympy.Symbol] = {}
-            self.symbol_name_to_range = {} if symbol_name_to_range is None else symbol_name_to_range
             self.signature = self.deserialize_signature(serialized_graph_module.signature)
             self.constants = deserialize_torch_artifact(constants)
+
+            # deserialization does analysis with checks on 0/1, so we create fake range constraints and
```

Comment:
where does it do this analysis, and is it not possible to avoid that instead of tweaking this logic just so?